Resources/Prototypes/Recipes/Lathes/medical.yml

```diff
@@ -173,3 +173,11 @@
   completetime: 2
   materials:
     Plastic: 300
+
+- type: latheRecipe
+  id: FelinidCone
+  result: FelinidCone
+  completetime: 2
+  materials:
+    Steel: 50
```

Comment:
I was going to say you could use plastic for this, until I read [this](https://en.wikipedia.org/wiki/Elizabethan_collar#Dangers_of_Elizabethan_collars).

Reply:
in this case, it is a plastic cone